tests/project/integration/app/README.md

```diff
@@ -0,0 +1,4 @@
+# README
```

Comment:
We have project already under `tests/test_data/projects`

Reply:
Hmm, I had put projects in this directory many weeks ago (for the initial project def implementation), is this the new home? If this is where you'd rather have all of them, would you like them moved in this PR?

Reply:
Yes, please let's move all projects under `tests_data`, we are using this place for long time. Sorry we haven't spotted this earlier